client/dashboard/src/pages/toolsets/Toolset.tsx

```diff
@@ -599,6 +601,14 @@ export function ToolsetView({
             />
           )}
         </TabsContent>
+        <TabsContent value="resources">
```

Comment:
Any chance we want to feature flag this or not show it for now if the user has no resources? Im not sure

Reply:
Ehh I kind of think having the tab now is not such a bad thing, a little teaser